rosdep/python.yaml

```diff
@@ -570,6 +570,13 @@ python-boltons:
     zesty:
       pip:
         packages: [boltons]
+python-boto3:
+  arch: [python-boto3]
+  debian: [python-boto3]
+  fedora: [python2-boto3]
+  gentoo: [dev-python/boto3]
+  opensuse: [python-boto3]
+  ubuntu: [python-boto3]
```

Comment:
Looks like this is new in Xenial, but we require support for Trusty still. Please use the `'*'` syntax and `null` for trusty, see:

https://github.com/ros-infrastructure/rep/blob/master/rep-0111.rst#allow-package_argument-to-be-explicitly-null